src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -262,27 +262,31 @@
    environment :- schema/Str
    jruby-service :- (schema/protocol jruby-protocol/JRubyPuppetService)
    request-tag :- (schema/maybe String)
-   last-updated :- (schema/maybe schema/Int)]
+   last-updated :- (schema/maybe schema/Int)
+   environment-class-cache-enabled :- schema/Bool]
```

Comment:
Added some more info to the docstring in [3317ef](https://github.com/camlow325/puppet-server/commit/3317ef2a5d56a0ff3eab905c7318a6a225511125) in #930.